synapse/handlers/oidc_handler.py

```diff
@@ -838,10 +817,14 @@ def _verify_expiry(self, caveat: str) -> bool:
         now = self.clock.time_msec()
         return now < expiry
 
-    async def _map_userinfo_to_user(
-        self, userinfo: UserInfo, token: Token, user_agent: str, ip_address: str
-    ) -> str:
-        """Maps a UserInfo object to a mxid.
+    async def _complete_oidc_login(
+        self,
+        userinfo: UserInfo,
+        token: Token,
+        request: SynapseRequest,
+        client_redirect_url: str,
+    ):
```

Comment:
I think we've been explicitly putting `None` as a return type otherwise mypy doesn't check something correctly? I don't remember the exact failure mode though.

Reply:
:+1: